static/app/views/starfish/views/spanSummaryPage/sampleList/sampleInfo/index.tsx

```diff
@@ -28,13 +30,23 @@ function SampleInfo(props: Props) {
     'span-summary-panel-metrics'
   );
 
+  const style: CSSProperties = {
+    textAlign: 'left',
```

Comment:
Left? Not right?

Reply:
Yeah, that way the number is actually under the `throughput` and `Duration (p95)` unlike rn
<img width="711" alt="image" src="https://github.com/getsentry/sentry/assets/44422760/0d0eb2d2-2d64-497f-8391-909dfe07e520">

Reply:
Oh my God, cells are used outside of tables, right, I forgot. Oof.

Reply:
Yup lol, It was done that way to keep all formatting consistent `(/s)`, can definitely do something else going forward.